
// let aa = null
console.log(aa)
try {
	aa = 10
	const aa = 22
} catch (e) {

}

let aa = 1
console.log(aa)